exercise/1-Python-Pandas/1-a-Python.ipynb

```diff
@@ -68,20 +68,20 @@
    "source": [
     "## Basic Data Types\n",
     "\n",
-    "- Numbers: integer, float, complex\n",
-    "- String `\"I'm a string\"`\n",
+    "- Number: integer, float, complex\n",
```

Comment:
Keep "Numbers" as you can have these three distinct types of numbers. Alternatively, you may call them "Numerical" if you want to stick with singular naming.